app/src/renderer/vuex/modules/session.js

```diff
@@ -186,41 +185,58 @@ export default () => {
       state.localKeyPairName = null
       commit(`setUserAddress`, null)
     },
-    loadErrorCollection({ state, dispatch }, address) {
-      const errorCollection =
-        localStorage.getItem(`${ERROR_COLLECTION_KEY}_${address}`) === `true`
+    loadLocalPreferences({ state, dispatch }) {
+      const localPreferences = localStorage.getItem(USER_PREFERENCES_KEY)
+
+      if (!localPreferences) {
+        state.cookiesAccepted = false
```

Comment:
use `commit`

Reply:
personally I have the feeling this just generates more code as the mutation is just setting this value nothing else. why do you think a commit here is valuable?

Reply:
oh I was just following our coding standards, but happy to discuss it with the team